grpcgcp/grpc_gcp/grpc_gcp.proto

```diff
@@ -73,6 +73,9 @@ message ChannelPoolConfig {
   // connection will not be interrupted. The unresponsive_detection_ms will be doubled every consecutive refresh
   // if no response from the server is received.
   uint32 unresponsive_calls = 7;
+
+  // Enables round-robin for binding calls.
+  bool round_robin_bind = 8;
```

Comment:
Are we doing transactional binding? Could you help to clarify the context we provide this option.
My understanding is:
- when starting a session/transaction, the first call would be bound to a channel in the pool, round robin may happen here.
- further calls belongs to the same session/transaction should use the same channel, this provides the stickiness on session/transaction level, and no round robin here.
- different picking policy maybe added later, such as least session/transaction, choose the channel with least session/transaction bound on it, or least load, choose the channel with least inflight requests, etc.

Reply:
> * when starting a session/transaction, the first call would be bound to a channel in the pool, round robin may happen here.

Correct

> * further calls belongs to the same session/transaction should use the same channel, this provides the stickiness on session/transaction level, and no round robin here.

Correct

> * different picking policy maybe added later, such as least session/transaction, choose the channel with least session/transaction bound on it, or least load, choose the channel with least inflight requests, etc.

Yes. Currently it is least load (least inflight requests) or round-robin. Do you suggest making it an enum now to account for possible new options in future?

Reply:
Converted to enum.